.dockerignore

```diff
@@ -1,7 +1,5 @@
-.git*
+.git
```

Comment:
git is never used to tag releases? (git hash in app or something?)

Reply:
not in the docker build